src/React/Containers/OverlayWindow/index.js

```diff
@@ -0,0 +1,414 @@
+        break;
+      case 'lrCorner':
+        this.dragHandler = this.handlerMap.bottomRight;
+        this.setState({ cursor: 'nwse-resize', dragging: true });
+        break;
+      default:
+        break;
+    }
+  },
+
+  mouseMove(evt) {
+    const actionStruct = this.computeActionRegion(evt);
+    this.setState({ cursor: actionStruct.cursor });
+  },
+
+  mouseDown(evt) {
+    const actionStruct = this.computeActionRegion(evt);
+    this.dragHandler = actionStruct.dragAction;
+    this.setState({ cursor: actionStruct.cursor, dragging: true });
+  },
+
+  mouseUp(evt) {
+    const actionStruct = this.computeActionRegion(evt);
+    this.dragHandler = this.mouseMove;
+    this.setState({ cursor: actionStruct.cursor, dragging: false });
+  },
+
+  render() {
+    if (!this.props.visible) {
+      return null;
+    }
+
+    // Configure the initial event container props and style overrides
+    const eventDivProps = {
+      className: style.eventContainer,
+      ref: 'eventContainerDiv',
+      style: {
+        pointerEvents: 'none',
+      },
+    };
+
+    // Configure the initial main container props and style overrides
+    const mainDivProps = {
+      className: style.mainContainer,
+      ref: 'mainContainerDiv',
+      style: {
+        pointerEvents: 'none',
+        width: this.state.width,
+        height: this.state.height,
+        top: this.state.y,
+        left: this.state.x,
+      },
+    };
+
+    // Make adjustments based on whether or not we're currently dragging
+    if (this.state.dragging === true) {
+      eventDivProps.onMouseUp = this.mouseUp;
+      eventDivProps.onMouseMove = this.dragHandler;
+      eventDivProps.style.pointerEvents = 'auto';
+      if (this.state.cursor !== null) {
+        eventDivProps.style.cursor = this.state.cursor;
+      }
+    } else {
+      mainDivProps.onMouseDown = this.mouseDown;
+      mainDivProps.onMouseUp = this.mouseUp;
+      mainDivProps.onMouseMove = this.dragHandler;
+      mainDivProps.style.pointerEvents = 'visible';
+      if (this.state.cursor !== null) {
+        mainDivProps.style.cursor = this.state.cursor;
+      }
+    }
+
+    // Configure the content container props and style overrides
+    const contentDivProps = {
+      className: style.content,
+      style: {
+        top: this.props.marginSize + this.props.titleBarHeight,
+        right: this.props.marginSize,
+        bottom: this.props.marginSize,
+        left: this.props.marginSize,
+      },
+    };
+
+    // Configure the title bar props and style overrides
+    const titleBarProps = {
+      className: style.titleBar,
+      style: {
+        top: this.props.marginSize,
+        right: this.props.marginSize,
+        left: this.props.marginSize,
+        height: this.props.titleBarHeight,
+        lineHeight: `${this.props.titleBarHeight}px`,
+      },
+    };
+
+    // Configure the hot corner divs
+    const off = this.props.hotCornerExtra;
```

Comment:
I'm ok with that.  I guess you figured out the reason for the short-lived variable: four lines in the jsx section below got a little unwieldy.

Thanks for taking a look :-)  I'll show you the context next time we hangout.